### ISSUE TITLE
getting user info CZECH

### ISSUE THREAD

**Issue by MichaelKazda** [MichaelKazda/PackMan-TikTok#1]
**Task:** The idea is to scrape an user from each network (all data you can get, if there is a lot of media, get only 10)

**Comment by MichaelKazda** [comment]
I wasnt able to get link from the same http request as the rest of informations, cos the part with videos is loaded afterwards.

I had to find another http request which respond is Json file, where are included video links.

But the script depends on the link with Json request and isnt possible find this link by itself, because the link contains GET parameters, such as _signature, which I am not able to find out how is it going to change for e.g. different user or more videolinks.

**BASICALLY it gets top 10 video links, but the link for the json must be find manually**
